ansible/roles/bastion-base/tasks/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - name: Setup Red Hat packages for bastion
+  when: not bastion_install_basic_packages | default(false)
```

Comment:
iirc, should have a `bool` filter?
```
  when: not bastion_install_basic_packages | default(false) | bool
```

Reply:
Actually... please set the default in the role defaults. Should not rely on an inline default and putting it in `defaults/main.yml` documents the variable.

Reply:
Good points both - I was surprised by the lack of a defaults, the good old days...